objectherkenning_openbare_ruimte/data_delivery_pipeline/components/data_delivery.py

```diff
@@ -101,6 +116,20 @@ def _create_filtered_metadata_files(
                     f"Exception during the creation of metadata file for: {video_name}: {e}"
                 )
 
+    @staticmethod
+    def _get_frame_numbers_in_metadata_file(file_path):
+        frame_numbers = []
+        with open(file_path) as metadata_file:
```

Comment:
for consistency with databricks, can we rename "metadata" into "detection_metadata" or "frame_metadata"?